doc/rados/configuration/ceph-conf.rst

```diff
@@ -599,16 +598,16 @@ Override values can be set in two ways:
 
       ceph tell osd.123 config set debug_osd 20
 
-   The `tell` command can also accept a wildcard for the daemon
-   identifier.  For example, to adjust the debug level on all OSD
-   daemons:
+   The ``tell`` command can also accept a wildcard as the daemon identifier.
+   For example, to adjust the debug level on all OSD daemons, run a command of
```

Comment:
Some might say that `OSD daemon` is redundant, but the usage is overloaded and entrenched.

Reply:
Yeah, Sage and I had a private email discussion about this. Because this term became a battleground about nine years ago and there was what Sage referred to as an industry and marketing push to redefine it, I'm going to fight to keep "daemon" around.